index.js

```diff
@@ -2016,9 +2058,7 @@
   //. > S.parseJson('[')
   //. Nothing()
   //. ```
-  S.parseJson = def('parseJson', [String], encase(function(s) {
-    return JSON.parse(s);
-  }));
+  S.parseJson = def('parseJson', [String], encase(JSON.parse));
```

Comment:
It'll be nice to no longer be dependent on the `length` of the provided function. :)